core/node/state_keeper/src/io/mempool.rs

```diff
@@ -309,6 +312,7 @@ impl StateKeeperIO for MempoolIO {
             timestamp,
             // This value is effectively ignored by the protocol.
             virtual_blocks: 1,
+            interop_roots: vec![],
```

Comment:
It feels, that it's better to move getting interop roots here. 
We discussed it with @kelemeno. 
But don't remember the exact outcome. 

Just because of it, we have this undesired behaviour in EN and it'd harder to sequence the action between during en node syncing. 
let's try to get rid of `load_latest_interop_root` and prepare the block params with interop roots inside the `wait_for_new_l2_block_params`